app/models.py

```diff
@@ -109,8 +131,8 @@ def serialize(self):
             'id': self.supplier_id,
             'name': self.name,
             'description': self.description,
-            # 'dunsNumber': self.duns_number,
-            # 'eSourcingId': self.esourcing_id,
+            'dunsNumber': self.duns_number,
```

Comment:
Not returning them complicates things: we accept them on PUT and they're part of the supplier schema, so it's reasonable to accept them on POST. But if they're not returned from the API then there's no indication if they've been stored or not and the data update can't use the `.serialize` since it will clear out the hidden fields. We should either drop the fields completely or treat them as normal fields, this "you can write, but can't read" seems to make little sense for the private API.